client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.html

```diff
@@ -493,7 +493,10 @@ <h3 data-translate="telephony_line_phone_order_summary"></h3>
                 </p>
                 <p data-ng-if="PhoneOrderCtrl.phone">
                     <strong data-translate="telephony_line_phone_order_return_address_title"></strong><br>
-                    <span data-translate="telephony_line_phone_order_return_address_content"></span>
+                    <span data-translate="telephony_line_phone_order_return_address_content_name"></span><br>
```

Comment:
To remove br tag, you can, (and apply this to `strong` tag as well) :
```suggestion
                    <span class="d-block" data-translate="telephony_line_phone_order_return_address_content_name"></span>
```

Reply:
Or 
```suggestion
                    <p data-translate="telephony_line_phone_order_return_address_content_name"></p>
```

Reply:
`<p>` tags have margin by default so going with `<span>`